### ISSUE TITLE
关于代码

### ISSUE THREAD

**Issue by suntea233** [yangli-hub/CMMT-Code#4]
`c = torch.sum(aux_output * aux_output, dim=-1)`
`gate_value = c.unsqueeze(-1)`
`reverse_gate_value = torch.neg(gate_value).add(1)`

c是由两个aux_output点乘，请问这里的c有什么意义呢

**Comment by yangli-hub** [comment]
hi, this part implements the Dynamic Multimodal Gate that introduced in section 4.4 of the paper.
